ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -338,6 +336,9 @@ def __float__(
     ):
         return ivy.array(ivy.reshape(self._ivy_array, -1), dtype=ivy.float64)[0]
 
+    def __str__(self):
```

Comment:
As I mentioned [here](https://github.com/unifyai/ivy/pull/8394#discussion_r1054456166), you need to recreate the function behaviour so that they match :)

Reply:
Hey @jkeane508 !
If you see the latest code, i have done the following
`return str(ivy.to_list(self._ivy_array))` for __str__ and it gives the correct format 

can you have another look at this and let me know